docs/guide/apm-getting-started.asciidoc

```diff
@@ -134,7 +134,7 @@ The HTTP API exposed by APM Server listens on `localhost` and port 8200 by defau
 If you change the listen address from `localhost` to something that is accessible from outside of the machine,
 we recommend setting up firewall rules to ensure that only your own systems can access the API.
 Alternatively,
-you can use the {apm-server-ref}/security.html[secret token and TLS] to secure access to APM Server API.
+you can use the {apm-server-ref}/overview.html[secret token and TLS] to secure access to APM Server API.
```

Comment:
@ruflin do you know if it would also work to use `{apm-server-ref}/overview.html#security[secret token and TLS] ` ?

Reply:
yes you can do anchor links. I'm doing that in the Node docs